protected/views/site/guide.php

```diff
@@ -23,75 +12,86 @@
-                                    <li><a href="/site/guidesoftware">Software Dataset Checklist</a></li>
-                                </ul>
-                            </li>
-                </div>
-            </section>
+        <?php
+            $this->widget('application.components.GuideNavigation');
+        ?>
             <section>
                 <div class="tab-content">
-                    <div role="tabpanel" class="tab-pane active" id="general">
-                        <h4 class="page-subtitle">General Submission Guidelines </h4>
+                    <div class="tab-pane active">
+                        <h2 class="page-subtitle h4">General Submission Guidelines </h2>
                         <div class="subsection">
                             <p>GigaDB is a <a target="_blank" href="https://www.cngb.org/aboutUs.html?i18nlang=en_US">China National GeneBank</a> supported repository used to host data and tools associated with articles in <i>GigaScience</i>. As part of your manuscript submission and in line with the <a target="_blank" href="https://academic.oup.com/gigascience/pages/editorial_policies_and_reporting_standards">Reporting Standards</a> and <a target="_blank" href="http://doi.org/10.25504/fairsharing.prdtva">FAIRsharing guidelines for data deposition and formatting for papers submitted to <i>GigaScience</i></a> we will provide an associated GigaDB dataset to host the data and files required for transparency and reproducibility. GigaDB is an open-access database. As such, all data submitted to GigaDB must be fully consented for public release (for more information about our data policies, please see our <a href="/site/term">Terms of use</a> page).
                             </p>
                         </div>
 
-                        <h4 class="page-subtitle">Workflow</h4>
+                        <h2 class="page-subtitle h4">Workflow</h2>
                         <div class="subsection">
                             <p>The workflow diagram below details a standard submission process:</p>
-                            <img src="/images/workflow.png" alt="Workflow"></img>
-                            <p>When contacted by curators to process the GigaDB dataset you will be invited to:<br>
-                                -	Create a GigaDB user account<br>
-                                -	Upload your prepared data files* (if not already public)<br>
-                                *- see checklists below<br>
-                                -	Supply the appropriate metadata<br>
-                                -	Proofread and approve the GigaDB pre-publication dataset page<br>
-                            </p>
+                            <figure>
+                                <img src="/images/workflow.png" alt="Workflow diagram of manuscript and data submission process for GigaScience, described in detail below"></img>
+                                <figcaption>
```

Comment:
text within figcaption is new content. It is essentially a transcription of the content of the diagram